snap/hooks/post-refresh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+SNAP_DATA=/var/snap/charmed-mysql/current
+
+# Ensure correct permissions on snap refresh
+chown -R 584788:root $SNAP_COMMON
+chown -R 584788:root $SNAP_DATA
```

Comment:
SNAP_DATA is defined above, but SNAP_COMMON is not.
Are there some special logic? Can we be consistent here?

Reply:
Apparently there's no good reason. It's fixed